core/controllers/library.py

```diff
@@ -308,20 +308,21 @@ def get(self):
 
 
 class CollectionSummariesHandler(base.BaseHandler):
-    """Returns Collection summaries corresponding to Collection ids.
+    """Returns collection summaries corresponding to collection ids.
     """
 
     GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
 
     def get(self):
         """Handles GET requests."""
         try:
-            coll_ids = json.loads(self.request.get('stringified_coll_ids'))
+            collection_ids = json.loads(
+                self.request.get('stringified_collection_ids'))
         except Exception:
             raise self.PageNotFoundException
         summaries = (
             summary_services.get_displayable_collection_summary_dicts_matching_ids(
-                coll_ids))
+                collection_ids))
```

Comment:
@seanlip, How can I fix the indentation here? The function is too long and I'm not able to break it.